.travis.yml

```diff
@@ -1,31 +1,44 @@
 ---
 sudo: false
+dist: trusty
 language: ruby
 cache: bundler
-script: "bundle exec rake release_checks"
-#Inserting below due to the following issue: https://github.com/travis-ci/travis-ci/issues/3531#issuecomment-88311203
 before_install:
-  - gem update bundler
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
+script:
+  - 'bundle exec rake $CHECK'
```

Comment:
Hmm, do we need the single quotes, and regardless, wonder if we should quote the CHECK expansion as `"$CHECK"`, just to avoid having to worry about the content.